cmd/repo-init/api.go

```diff
@@ -228,10 +234,10 @@ func authHandler(log *logrus.Entry) http.HandlerFunc {
 	}
 }
 
-func clusterProfileHandler(log *logrus.Entry) http.HandlerFunc {
+func (s *server) clusterProfileHandler() http.HandlerFunc {
```

Comment:
I feel handler funcs should not be attached to the server.

Reply:
I may be missing something, but I think that is exactly what @droslean was suggesting I do in https://github.com/openshift/ci-tools/pull/2672#pullrequestreview-882975470.

Reply:
I get the logger part, but not the handler part: the functions do not use anything from the server's objects.
I guess no big deal except personal taste.